components/tidb_query/src/batch/runner.rs

```diff
@@ -46,6 +47,12 @@ pub struct BatchExecutorsRunner<SS> {
     collect_exec_summary: bool,
 
     exec_stats: ExecuteStats,
+
+    /// Minimum rows to return in batch stream mode.
+    stream_min_rows_each_iter: usize,
+
+    /// `batch_size` in strReam mode, this variable will be initialized as `BATCH_INITIAL_SIZE`.
```

Comment:
typo: strReam

Reply:
At the time reading this comment, there is no `batch_size`. So you still need to explain what is `batch_size`, or at least give some reference.